Dockerfile

```diff
@@ -28,16 +29,40 @@ FROM base as builder
 RUN apk --update --no-cache add $BUILD_DEPS
 
 # create venv
-RUN pip install -U pip pipenv
+RUN pip install -U pip wheel
+
+# pipenv install
+# RUN pip install -U pipenv
+
+# poetry install
+RUN pip install "poetry==$POETRY_VERSION"
 RUN python3 -m venv "$VIRTUAL_ENV"
 
 WORKDIR /build
+
+# PipFile install
+# COPY Pipfile* ./
+# RUN pipenv install --dev --ignore-pipfile
+
 # individual files here instead of COPY . . for caching
-COPY Pipfile* ./
-COPY Ctl/VERSION Ctl/
-RUN pipenv install --dev --ignore-pipfile
+COPY pyproject.toml poetry.lock ./
+
+RUN . "$VIRTUAL_ENV"/bin/activate && poetry install --no-root
+
+COPY . .
+RUN . "$VIRTUAL_ENV"/bin/activate && poetry build
```

Comment:
Not building a package at all, so this line can probably go?

Reply:
Yes, taking it out.

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/7cb9e0cd7285023ca68f5d0f722153587a816fb3